airbyte-container-orchestrator/src/main/resources/application.yml

```diff
@@ -3,4 +3,18 @@ micronaut:
     port: 9000
 
 airbyte:
-  config-dir: /config
\ No newline at end of file
+  config-dir: /config
+  control:
+    plane:
+      auth-endpoint: ${CONTROL_PLANE_AUTH_ENDPOINT:}
+  data:
+    plane:
+      service-account:
+        credentials-path: ${DATA_PLANE_SERVICE_ACCOUNT_CREDENTIALS_PATH:}
+        email: ${DATA_PLANE_SERVICE_ACCOUNT_EMAIL:}
+  internal:
+    api:
+      auth-header:
+        name: ${AIRBYTE_API_AUTH_HEADER_NAME:}
+        value: ${AIRBYTE_API_AUTH_HEADER_VALUE:}
+      host: ${INTERNAL_API_HOST}
```

Comment:
This probably means we need to add more environment variables to the container-orchestrator. I'm in the process of trying this out on a dev cloud environment and will make a PR over there to add the env vars as needed. Haven't been able to fully test due to some incompatibility with the latest OSS branch contents and Cloud that are currently being worked through.

Reply:
See https://github.com/airbytehq/airbyte/pull/20724 and https://github.com/airbytehq/airbyte-cloud/pull/3902